io.edgehog.devicemanager.ForwarderSessionsState.json

```diff
@@ -0,0 +1,15 @@
+{
+  "interface_name": "io.edgehog.devicemanager.ForwarderSessionsState",
+  "version_major": 0,
+  "version_minor": 1,
+  "type": "properties",
+  "ownership": "device",
+  "description": "This property is set by a device to provide information about the status of a remote terminal session.",
+  "mappings": [
+    {
+      "endpoint": "/%{session_token}/connected",
+      "type": "boolean",
+      "description": "Indicates if the device is connected to a remote terminal session "
```

Comment:
I am not sure a boolean is enough.

I would suggest 3 different states:
- disconnected
- connecting
- connected

Right now if the device is returned a 4XX error code during the connection it will not try again and will consider the connection attempt as a failure.
If the device goes connecting -> disconnected, edgehog can infer that the connection has failed.
This is not possible with just a boolean value, as it would always appear as disconnected.

Reply:
Done